### PR TITLE
controllers drones and bookings generated + routes

### PR DESCRIPTION
**Pull request by SabrinaNayeli**
everything is in the title

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
```
Comment (AndyLukac) [inline config/routes.rb:3]: pages#home = PagesController, Action Home
We do not have a Pages controller so it should be DronesController (drones#home)

**Review by AndyLukac — COMMENTED** [review]
Please verify.

File: config/routes.rb (hunk 1 above)
Comment (Tom-Firth) [reply config/routes.rb:3]: You are saying n'importe quoi Andy, look in your app/controllers file ;)

Comment (AndyLukac) [reply config/routes.rb:3]: desoleeeeeeeee

**Pull request merged**